SearchStrings.java

```diff
@@ -0,0 +1,67 @@
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+public class SearchStrings {
+	int count = 0;
+	int lineNumber = 0;
```

Comment:
Is there a reason this is not simply a local variable in `stringCount`? If possible, we could just make everything `static`